docs/README.md

```diff
@@ -16,14 +16,17 @@ Installation
 Install it in the conda environment
 
 ```bash
-conda install -c anaconda sphinx
-conda install -c anaconda sphinx_rtd_theme
-conda install -c conda-forge sphinxcontrib-programoutput
+conda install -c conda-forge sphinxcontrib-programoutput sphinx sphinx_rtd_theme
 ```
 
 Running
 -------
-```make html```
+
+```bash
+make clean html
+make html
```

Comment:
I don't think we need `make html` if we are already doing `make clean html`. `make clean html` executes `make html` and cleans `_build` as well.